openfisca_core/entities/tests/test_entity.py

```diff
@@ -0,0 +1,11 @@
+from openfisca_core import entities
+
+
+def test_init_when_doc_indented() -> None:
+    """De-indent the ``doc`` attribute if it is passed at initialisation."""
+
+    key = "\tkey"
+    doc = "\tdoc"
+    entity = entities.Entity(key, "label", "plural", doc)
+    assert entity.key == key
+    assert entity.doc != doc
```

Comment:
This does not test that there is de-indentation, but that the strings are different. This approximation is too broad and can give false negatvies too easily. What about:

```suggestion
    assert group_entity.doc == doc.lstrip()
```